src/builders/browser/index.ts

```diff
@@ -16,8 +17,15 @@ export class SingleSpaBrowserBuilder extends BrowserBuilder {
     root: Path,
     projectRoot: Path,
     host: virtualFs.Host<fs.Stats>,
-    options: SingleSpaBuilderSchema
+    options: SingleSpaBuilderSchema,
   ): Configuration {
+
+    const libraryName = options.libraryName || this.getProjectName();
+
+    if (!libraryName) {
+      throw new Error('Unable to determine libraryName.');
```

Comment:
In webpack, `output.library` is not required for all possible values of `output.libraryTarget`. Specifically, the `system` libraryTarget does not require a name, and actually SystemJS@3 does not support giving it one. So I think we should remove this error check. Instead we should let webpack do it's validation on the webpack config, which will include throwing an error if the libraryName is needed but not defined.